servers/rendering/renderer_rd/shaders/scene_forward_lights_inc.glsl

```diff
@@ -944,7 +944,7 @@ void reflection_process(uint ref_index, vec3 vertex, vec3 normal, float roughnes
 
 		vec4 reflection;
 
-		reflection.rgb = textureLod(samplerCubeArray(reflection_atlas, material_samplers[SAMPLER_LINEAR_WITH_MIPMAPS_CLAMP]), vec4(local_ref_vec, reflections.data[ref_index].index), roughness * MAX_ROUGHNESS_LOD).rgb;
+		reflection.rgb = textureLod(samplerCubeArray(reflection_atlas, material_samplers[SAMPLER_LINEAR_WITH_MIPMAPS_CLAMP]), vec4(local_ref_vec, reflections.data[ref_index].index), roughness * MAX_ROUGHNESS_LOD).rgb * scene_data.color_scale;
```

Comment:
I would probably make this a specialization constant in the mobile renderer and a regular constant (1.0) in the clustered one.

Reply:
See #51635 for reference (which I guess should be merged first).

Reply:
after merging that PR, on line 400 of shader_forward_mobile.glsl you can add:

`layout(constant_id = 15) const float sc_luminance_multiplier = 2.0;`

This has two advantages. First is that you can also use it to divide at the end of the shader (and it will be turned to a multiplication by the compiler). Second, that in the clustered one you can simply do something like:

`const float sc_luminance_multiplier = 1.0;`

and this will be optimized away.

Reply:
Ok I'll look into that, need to learn how that works :)